imports/ui/JapaneseNewGame.js

```diff
@@ -371,6 +369,7 @@ Template.JapaneseNewGame.events({
 						if (NewGameUtils.noIllegalJapaneseHands()) {
 							push_dealin_hand(template);
 							$( ".delete_hand_button" ).removeClass( "disabled" );
+							resetRoundStats();
```

Comment:
I'm starting to wonder if this section should be refactored in such a way that this duplication is unnecessary.  I worry about one of these blocks being missed whenever there is so much duplication.